src/mitol/payment_gateway/api.py

```diff
@@ -361,6 +431,47 @@ def prepare_checkout(
             "method": "POST",
         }
 
+    def get_client_configuration(self):
+
+        configuration_dictionary = ({})
+        configuration_dictionary["authentication_type"] = "http_signature"
+        configuration_dictionary["merchantid"] = settings.MITOL_PAYMENT_GATEWAY_CYBERSOURCE_MERCHANT_ID
+        if settings.MITX_ONLINE_ENVIRONMENT == 'prod':
+            configuration_dictionary["run_environment"] = "apitest.cybersource.com"
+        else:
+            configuration_dictionary["run_environment"] = "api.cybersource.com"
+        configuration_dictionary["merchant_keyid"] = settings.MITOL_PAYMENT_GATEWAY_CYBERSOURCE_MERCHANT_SECRET_KEY_ID
+        configuration_dictionary["merchant_secretkey"] = settings.MITOL_PAYMENT_GATEWAY_CYBERSOURCE_MERCHANT_SECRET
+        configuration_dictionary["timeout"] = 1000
+        return configuration_dictionary
```

Comment:
Suggestion: you should be able to inline all of this into a single dict definition

Reply:
Yes, The PR is in a rough state yet but it will be cleaner once finalized.